### ISSUE TITLE
Undo disable-dnscache-service-win.bat

### ISSUE THREAD

**Issue by mrhobbeys** [StevenBlack/hosts#2508]
I'm struggling with a DNS problem after disabling the DNScache. I already set: "HKLM\SYSTEM\CurrentControlSet\services\Dnscache" /v Start /t REG_DWORD /d 4 /f 

to 

"HKLM\SYSTEM\CurrentControlSet\services\Dnscache" /v Start /t REG_DWORD /d 2 /f

However, I can't get the computer to contact the domain or start the DNS service.

**Comment by mrhobbeys** [comment]
I believe this ticket can now be closed as I have provided the correct method to "undo" the changes made by the bat file in my first request. 

At the time of my original message, the server had numerous issues, which made it difficult for me to identify the correct solution. It is worth noting that even after changing the registry setting as advised, the server still exhibited some persistent issues. Fortunately, by making the DNS change when I did, I was able to identify other issues that had been waiting to surface. The server had lost the ability to communicate with itself due to cloning and moving. Though the DNS cache was allowing it to function and get by, I suspect that any network diagnostics or DNS changes would have caused a complete failure.

**Comment by mrhobbeys** [comment]
I believe this ticket can now be closed as I have provided the correct method to "undo" the changes made by the bat file in my first request. 

At the time of my original message, the server had numerous issues, which made it difficult for me to identify the correct solution. It is worth noting that even after changing the registry setting as advised, the server still exhibited some persistent issues. Fortunately, by making the DNS change when I did, I was able to identify other issues that had been waiting to surface. The server had lost the ability to communicate with itself due to cloning and moving. Though the DNS cache was allowing it to function and get by, I suspect that any network diagnostics or DNS changes would have caused a complete failure.
